docs/spec_draft.md

```diff
@@ -670,6 +717,54 @@ operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.remainder
+
+`stablehlo.remainder(dividend, divisor) -> result`
+
+### Semantics
+
+Performs element-wise remainder of tensors `dividend` and `divisor` and produces
+a `result` tensor. The sign of the result is taken from the dividend, and the
+absolute value of the result is always less than the divisor's absolute value.
+For floating-point element types, remainder of the any division operation `x/y`
+is calculated as `x - n*y`, where `n` is `x/y` with its fractional part
+truncated. The corner cases are TBD. For n-bit integer, division overflow
```

Comment:
I think it would be useful to call out that this is different from IEEE-754 `remainder` and explain why.